schemas/compat-data-schema.md

```diff
@@ -258,16 +258,26 @@ Examples:
 
 #### `prefix`
 A prefix to add to the sub-feature name (defaults to empty string).
-Note that leading and trailing `-` must be included. Example:
+If applicable, leading and trailing `-` must be included.
 
-* Prefixed sub-feature:
+Examples:
+
+* Prefixed CSS sub-feature:
 ```json
 {
   "prefix": "-moz-",
   "version_added": "3.5"
 }
 ```
 
+* Prefixed API sub-feature:
```

Comment:
This'd probably be more clear with an example, e.g.

FeatureName with a vendor-prefixed implementation in WebKit, called webkitFeatureName would be represented like so:
{
  "prefix": "webkit",
  "version_added": "9"
}

Reply:
Good idea! I expanded both like this. Curious to know what you think